doc/specs/#1043 - Set the initial position of the Terminal/spec.md

```diff
@@ -9,15 +9,15 @@ issue id: #1043
 
 ## Abstract
 
-This spec is for task #1043 �Be able to set an initial position for the terminal�. It goes over the details of a new feature that allows users to set the initial position and size of the terminal. Expected behavior and design of this feature is included. Besides, future possible follow-up works are also addressed.
+This spec is for task #1043 “Be able to set an initial position for the terminal”. It goes over the details of a new feature that allows users to set the initial position and size of the terminal. Expected behavior and design of this feature is included. Besides, future possible follow-up works are also addressed.

```

Comment:
If anyone wonders what's different here: The old/current file is encoded as CP 1252 and this PR encodes it as UTF-8.
In my opinion you can feel absolutely free to just replace these uncommon Unicode quotes and dashes with standard ASCII ones. None of the other doc files uses those anyways.

Reply:
![screen](https://user-images.githubusercontent.com/53474591/120803037-bdfdef00-c560-11eb-927b-325ba829d123.png)
yeah, I can see them here...